CHANGES

```diff
@@ -1,4 +1,4 @@
-2.0~beta
+2.0~alpha4
```

Comment:
On second thought that should cause you no trouble if you were already on alpha3 (the format didn't change in between, so no upgrade should have been performed, and the `opam-version` field is left unchanged in that case)